models/demos/wormhole/stable_diffusion/tt/ttnn_functional_resnetblock2d_new_conv.py

```diff
@@ -409,7 +409,7 @@ def __call__(
                 weights_dtype=ttnn.bfloat8_b,
                 math_fidelity=ttnn.MathFidelity.LoFi,
                 activation="",
-                height_sharding=False,
+                shard_layout=ttnn.TensorMemoryLayout.BLOCK_SHARDED,
```

Comment:
Is `height_sharding=False` functionally equivalent to `shard_layout=ttnn.TensorMemoryLayout.BLOCK_SHARDED`?

Reply:
Yes. Previously there was only height_sharding and block sharding. So height_sharding=False implied block_sharding.